types/express/express-tests.ts

```diff
@@ -71,6 +71,15 @@ namespace express_tests {
         language = req.acceptsLanguages(['en', 'ja']);
         language = req.acceptsLanguages('en', 'ja');
 
+        let existingHeader1 = req.get('existingHeader') as string;
```

Comment:
if you write
```
let header: string | undefined;
header = req.get('existingHeader');
```
and `req.get` returns `string` only, it will pass because `string` can be assigned to `string | undefined`

if you write what I did
```
req.get('existingHeader') as string;
req.get('existingHeader') as undefined;
```
and req.get returns just `string`, you'll get an error `type string cannot be converted to type undefiend` on second statement

this was the only way I managed to write tests for this change in a way that they would fail before updating definitions and pass after they're updated